accounts/abi/bind/backends/simulated_test.go

```diff
@@ -27,6 +27,8 @@ import (
 	"testing"
 	"time"
 
+	"go.uber.org/goleak"
```

Comment:
please don't add the whole goleak dependency, just the fix is fine

Reply:
Got it, I removed the dependency, and ran `go mod tidy`.

Reply:
Thanks, I rebased my branch with master branch.